### ISSUE TITLE
关于对query_bbox作inverse_sigmoid

### ISSUE THREAD

**Issue by kingofstu** [MCG-NJU/SparseBEV#85]
query_bbox初始化时xy坐标是做了归一化：

grid_size = int(math.sqrt(self.num_query))         # 900
assert grid_size * grid_size == self.num_query
x = y = torch.arange(grid_size)             # 30,30
xx, yy = torch.meshgrid(x, y, indexing='ij')  # [0, grid_size - 1]
xy = torch.cat([xx[..., None], yy[..., None]], dim=-1)
xy = (xy + 0.5) / grid_size  # [0.5, grid_size - 0.5] / grid_size ~= (0, 1)
with torch.no_grad():
    self.init_query_bbox.weight[:, :2] = xy.reshape(-1, 2)  # [Q, 2]

但为啥在refine_bbox时要inverse_sigmoid(bbox_proposal[..., 0:3]) ？

**Comment by afterthat97** [comment]
你总得回到真实坐标吧。。
